ospfd/ospf_ri.c

```diff
@@ -448,7 +448,7 @@ static void set_sr_global_label_range(struct sr_block srgb)
 	/* Set Header */
 	TLV_TYPE(OspfRI.sr_info.srgb) = htons(RI_SR_TLV_SRGB_LABEL_RANGE);
 	TLV_LEN(OspfRI.sr_info.srgb) =
-		htons(SUBTLV_SID_LABEL_SIZE + sizeof(uint32_t));
+		htons(RI_SR_TLV_LABEL_RANGE_SIZE + sizeof(uint32_t));
```

Comment:
This is not correct. SRGB or SRLB size is 12. Here, you set the length to 12 + 4 = 16 which is wrong. The correct line is:
``TLV_LEN(OspfRI.sr_info.srgb) = htons(RI_SR_TLV_LABEL_RANGE_SIZE);``

Reply:
Sure, sorry, my bad. Should be correct now. But it looks like the change doesn't actually fix anything, just making code more consistent.

Reply:
Yes. There is no modification that could fix anything. IMHO, there is a problem / bug in the code that depends on the compilation. I mean that you could have a stable code and just by changing some lines you produce a bad code. And the problem / bug is not always relate to the modification. I'm suspect a problem with pointer / memory allocation ... but where it is ? I don't know

Reply:
Anyway, let's merge this one because it fixes the `SUBTLV_SID_LABEL_SIZE`. It will matter if anytime later we start using this definition :)